bgpd/bgp_vty.c

```diff
@@ -10606,7 +10585,7 @@ DEFUN (show_ip_bgp_peer_groups,
 	vrf = argv_find(argv, argc, "WORD", &idx) ? argv[idx]->arg : NULL;
```

Comment:
looks like VRF will never be set as "WORD" not in command syntax.  What am I missing?

Reply:
yeah that's a bug